docs/api/config.md

```diff
@@ -6,6 +6,52 @@ Mesop is configured at the application level using environment variables.
 
 ## Configuration values
 
+### MESOP_STATIC_FOLDER
+
+> **NOTE:** By default, this feature is not enabled, but in an upcoming release, the
+default will be `static`.
+
+Allows access to static files from the Mesop server.
+
+It is important to know that the specified folder path is relative to the current
+working directory where the Mesop command is run. Absolute paths are not allowed.
+
+Example:
+
+In this case, the current working directory is `/srv`, which means Mesop will make
+`/srv/static` the static folder.
+
+```bash
+cd /srv
+MESOP_STATIC_FOLDER=static mesop app/main.py
+```
+
+Here are some examples of valid paths. Let's assume the current working directory is
+`/srv/`
+
+- `static` becomes `/srv/static`
+- `static/` becomes `/srv/static`
+- `static/assets` becomes `/srv/static/assets`
+- `./static` becomes `/srv/static`
+- `./static/` becomes `/srv/static`
+- `./static/assets` becomes `/srv/static/assets`
+- `.` becomes `/srv/static/.`
```

Comment:
I don't understand these last two examples. It's unclear how `.` becomes `/srv/static/.`

Also, I think prohibiting "." and "./" could be good because it seems fairly risky (and probably un-intended) to serve all your Mesop app code (e.g. py modules) - https://github.com/google/mesop/pull/989#discussion_r1812006176